nixos/modules/services/networking/prosody.nix

```
@@ -266,6 +266,15 @@ let
     else if builtins.isList x then "{ ${lib.concatMapStringsSep ", " toLua x} }"
     else throw "Invalid Lua value";
 
+  settingsToLua = prefix: settings: generators.toKeyValue {
+      listsAsDuplicateKeys = false;
+      mkKeyValue = k:
+        generators.mkKeyValueDefault {
+          mkValueString = toLua;
+        } " = " (prefix + k);
+    }
+    (filterAttrs (k: v: v != null) settings);
```

```suggestion
    listsAsDuplicateKeys = false;
    mkKeyValue = k: generators.mkKeyValueDefault {
      mkValueString = toLua;
    } " = " (prefix + k);
  } (filterAttrs (k: v: v != null) settings);
```
according to the contributing guide we are using 2 spaces